src/migrations/1692624998160-category-seed.ts

```diff
@@ -0,0 +1,102 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class Migrations1692624998160 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    const update = async (name: string, newName: string) => {
+      await queryRunner.query(`
+            UPDATE category
+            SET name='${newName}'
+            WHERE name='${name}'
+        `);
+    };
+
+    // update categories for language
+    await update('English', 'english');
+    await update('French', 'french');
+    await update('German', 'german');
+
+    // add new languages
+    await queryRunner.query(`INSERT INTO category (name, type) VALUES (
+        'arabic',
+        'language'
+        );
+    `);
+    await queryRunner.query(`INSERT INTO category (name, type) VALUES (
+        'italian',
+        'language'
+        );
+    `);
+
+    // update categories for discipline
+    await update('Arts', 'arts');
+    await update('Language', 'language');
+    await update('Math', 'mathematics');
+    await update('Literature', 'literature');
+    await update('Natural Science', 'natural-science');
+    await update('Social Science', 'social-science');
+
+    // update categories for level
+    await update('Kindergarden', 'kindergarten');
```

Comment:
Is the change from `kindergarden` to `kindergarten` intentional ?

Reply:
The `t` one is the correct one!